src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,123 @@
+@use "./variables" as *;
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(1, $card-width);
+    margin: auto;
+    gap: 46px 48px;
+    padding: 50px 40px;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
container is not part of card, you should have separate file for it

Reply:
separate file for container done